src/pptxgen.ts

```diff
@@ -220,7 +220,9 @@ export default class PptxGenJS {
         return this._presLayout
     }
 
-    constructor() {
+    constructor(
+        params: { presLayout?: string } = { presLayout: DEF_PRES_LAYOUT }
```

Comment:
Not sure this is what you want. Here it will only default if there is not params set. It should be
```.ts
{ presLayout?: string = DEF_PRES_LAYOUT } = {  }
```

(I think)

Reply:
This is exactly what I tried at first but for some reason TypeScript complier didn't agree with me. 😔

Reply:
<img width="1125" alt="Screenshot 2020-04-12 at 10 55 47" src="https://user-images.githubusercontent.com/56879/79064790-31350980-7cac-11ea-9c4a-f30cbb8fd70e.png">

Reply:
Just remove the type annotations it should work (yeah I know...)

Reply:
same thing 🙃
<img width="1063" alt="Screenshot 2020-04-12 at 13 30 43" src="https://user-images.githubusercontent.com/56879/79067733-df977980-7cc1-11ea-94d7-4b1d7fd6cd73.png">

Reply:
ok I thought I've tried all the options...
<img width="993" alt="Screenshot 2020-04-13 at 09 13 18" src="https://user-images.githubusercontent.com/56879/79100631-3c993b00-7d67-11ea-8bfb-0939a5014061.png">

Reply:
Do you understand how it is different?

Reply:
slightly different way of defining the constructor params https://stackoverflow.com/a/32596200